packages/spotlight/SpotlightRootDecorator/SpotlightRootDecorator.js

```diff
@@ -53,14 +64,26 @@ const defaultConfig = {
  * @hoc
  */
 const SpotlightRootDecorator = hoc(defaultConfig, (config, Wrapped) => {
-	const {noAutoFocus} = config;
+	const {getInputTypeSetter, noAutoFocus, rootId} = config;
+	const rootNode = typeof document === 'object' && document.querySelector('#' + rootId) || document;
+	const input = {
+		activated: false,
+		applied: false,
+		types: {
+			key: true,
+			mouse: false,
+			touch: false
+		}
```

Comment:
Is there a reason that it should be object, not just string?
types: 'key'

Reply:
To avoid human error, `map()` is used to add or remove class names rather than using separate sentences for each class name. Properties in the object are matched to each class name here. Please see the line 127~128.